### PR TITLE
Homepage/container

### PR DESCRIPTION
**Pull request by DlnyaMazhari**
# Description

home page components are added into its container and the page is responsive to various desktop sizes as well.

Fixes # (#8 )

# How to test?

npm run dev

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by Poorooshka — APPROVED** [review]
I tested the responsiveness of home page in the following resolutions: 
• 1920x1080 (19.57%)
• 1366x768 (14.88%)
• 1440x900 (9.18%)
• 1536x864 (7.22%)
• 1024x768 (4.73%)
and they were all responsive.

**Review by dpfernandes — APPROVED** [review]
👍

File: src/client/containers/Home/Home.js [hunk 1]
```diff
@@ -42,20 +42,25 @@ export const Home = () => {
...
-              size="big"
-              isMono={true}
-              color="orange"
-            />
+            <Link to={'/CareerPage'}>
+              <Buttons
+                label="Take the career quiz"
+                size="big"
+                isMono={true}
+                color="orange"
```
Comment (TITHI1244) [inline src/client/containers/Home/Home.js:50]: I think, we do not need to pass the color-orange props as the default is mentioned as orange :)

**Review by TITHI1244 — APPROVED** [review]
For styling units, em/vh/% were used instead of px and media queries also look super nice to me. Great job :)

File: src/client/containers/Home/Home.js [hunk 2]
```diff
@@ -42,20 +42,25 @@ export const Home = () => {
...
-              size="big"
-              isMono={true}
-              color="orange"
-            />
+            <Link to="/CareerPage">
+              <Buttons
+                label="Take the career quiz"
+                size="big"
+                isMono={true}
+                color="orange"
```
Comment (bytewiz) [inline src/client/containers/Home/Home.js:50]: This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an `<a>` tag around a `<button>` tag. 

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Comment (bytewiz) [reply src/client/containers/Home/Home.js:50]: If you want to navigate to a page using a button this is the correct way to do so :). 

https://reactrouter.com/web/api/Hooks/usehistory

File: src/client/containers/Home/Home.js [hunk 3]
```diff
@@ -42,20 +42,25 @@ export const Home = () => {
...
-              color="orange"
-            />
+            <Link to="/ElevatorPitchPage">
+              <Buttons
+                label="Visit the elevator"
+                size="big"
+                isMono={true}
+                color="orange"
+              />
+            </Link>
```
Comment (bytewiz) [inline src/client/containers/Home/Home.js:63]: Same here as above

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/containers/Home/Home.js (hunk 2 above)
Comment (DlnyaMazhari) [reply src/client/containers/Home/Home.js:50]: we changed

**Review by bytewiz — APPROVED** [review]
nice!

**Pull request merged**